### ISSUE TITLE
[3.9.x] [a11y] Number of positions selector is not accessible

### ISSUE THREAD

**Issue by zwiastunsw** [joomla/joomla-cms#24469]
### Steps to reproduce the issue
1. Use screen reader, e.g. NVDA
2. Go to the search results list page (for ex. use search module to search for a popular word, e.g. joomla on a test site)
3. Go to "Display #" list.
4. Select any number position

### Expected result
1. The screen reader announces the selected number of items
2. After pressing the Enter key, the page is reloaded and the selected number of items is displayed.

### Actual result
1. The screen reader not announces the selected number
2. After a while, the page is automatically reloaded.

### Note
The same error occurs on a **Tagged items** page type.

**Comment by ghost** [comment]
Can you please open a new Issue for the "Tagges Items"-Note so we have one Topic in one Issue?

**Comment by zwiastunsw** [comment]
OK

**Comment by zwiastunsw** [comment]
Solved - https://github.com/joomla/joomla-cms/pull/24623. Thanks @brianteeman
